check.py

```diff
@@ -413,6 +413,7 @@ def fix(x):
     print s
     f = open('a.js', 'w')
     f.write(open(os.path.join(options.binaryen_bin, 'binaryen.js')).read())
+    f.write('var Binaryen = module.exports;\n')
```

Comment:
As I am unable to run the tests locally because I am on Windows and the Travis build is also failing lately, things become a bit difficult here.

I assume that 'mozjs' is just a recent Spidermonkey shell, is that correct? If so, the version I am running locally doesn't seem to provide a CommonJS environment like the initial comment in `binaryen.js-post.js`, that lead me to adding the additional line to `check.py`, stated (there's no `module` or `require` defined). If the latter assumption is correct, the additional line added to `check.py` wouldn't be necessary, but hard to tell without a way to test this.